testsuite/oxigraph-tests/geosparql/sf_contains.rq

```diff
@@ -0,0 +1,9 @@
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX geof: <http://www.opengis.net/def/function/geosparql/>
+
+SELECT ?left ?right (geof:sfContains(?left, ?right) AS ?result) WHERE {
+    VALUES (?left ?right) {
+        ("Polygon((0 0, 0 3, 3 3, 3 0))"^^geo:wktLiteral "Point(1 2)"^^geo:wktLiteral)
```

Comment:
What about MULTIPOLYGON and POLYGON Z?

Reply:
`MULTIPOLYGON` definitely works and I need to check for `POLYGONZ` but [it seems supported by the WKT parser I use](https://github.com/georust/wkt/blob/b2f73990d0f0b3b800b09d7e868291dce4f3d263/src/lib.rs#L225) so I have good hope.